enterprise/admin/forms.py

```diff
@@ -501,3 +502,43 @@ class Meta:
             'decrypted_password': forms.widgets.PasswordInput(),
             'decrypted_sftp_password': forms.widgets.PasswordInput(),
         }
+
+
+class TransmitEnterpriseCoursesForm(forms.Form):
+    """
+    Form to transmit courses metadata for enterprise customers.
+    """
+    catalog_user = forms.CharField(
+        label=_('Enter enterprise worker username.'),
+        required=True
+    )
+
+    def __init__(self, *args, **kwargs):
+        """
+        Initializes form: puts current enterprise_customer into a field for
+        later access.
+
+        Arguments:
+            enterprise_customer (enterprise.models.EnterpriseCustomer): current customer
+        """
+        self._enterprise_customer = kwargs.pop('enterprise_customer', None)
```

Comment:
reason for removing the key from `kwargs` ?

Reply:
We setup enterprise customer from the detail view instead of getting the enterprise UUID from the user.

Reply:
but reason for `pop` instead of `get` ?